src/content/docs/dev/amplifier/gmp-api.mdx

```diff
@@ -0,0 +1,441 @@
+# GMP API
+
+The Axelar Amplifier [GMP (General Message Passing)](/dev/general-message-passing/overview/) API equips developers with comprehensive tools and documentation to interact with Axelar's Amplifier protocol. 
```

Comment:
Can you add some more context on when/why this API is to be used? This is an API for integrators, not for developers sending GMP messages, right?

Is this always needed, or is this part of relaying? Maybe we should make another subsection about relaying. How does this work if Interop Labs is doing relaying for you?

Reply:
Added more context to the top of the GMP API section.


I ended up doing a larger refactor just now as the "automatic" section of the relayer docs was really explaining the relayer itself not sure why it was in there and I instead added this GMP-API section to what was previously the "automatic relaying" section of the doc.

AFAIK the GMP API is used to simplify the process of triggering the Amplifier contracts to a matter of endpoints being called rather than using the `axleard` commands. So it's strongly recommended teams use this instead of going the manual route, assuming Interop Labs is not doing the relaying work for you.

Reply:
Resolving for now @StephenFluin but feel free to reopen if you think further discussion required here.